libparsec/src/invite.rs

```diff
@@ -111,7 +111,7 @@ pub async fn bootstrap_organization(
     sequester_authority_verify_key: Option<SequesterVerifyKeyDer>,
 ) -> Result<AvailableDevice, BootstrapOrganizationError> {
     let config: Arc<libparsec_client::ClientConfig> = config.into();
-    let events_plugged = OnEventCallbackPlugged::new(on_event_callback);
+    let events_plugged = OnEventCallbackPlugged::new(handle, on_event_callback);
```

Comment:
Always pass `0` for handle here (i.e. "always invalid handle"), and add a comment to explain it is the pragmatical approach

Reply:
done